mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -53,13 +53,13 @@ def __init__(self, package, port, adb_proxy):
     def _do_start_app(self):
         """Overrides superclass."""
         cmd = _LAUNCH_CMD.format(self._port, self._package)
-        logging.info('Launching snippet apk with: %s', cmd)
+        logging.debug('Launching snippet apk with: %s', cmd)
```

Comment:
I want this log on console. Since people have to create their own instrumentations and manifest, seeing exactly what the command is doing helps a lot with troubleshooting issues like missing instrumentation.